packages/lib/gpcircuits/scripts/common.ts

```diff
@@ -0,0 +1,170 @@
+import { Circomkit, CircomkitConfig } from "circomkit";
```

Comment:
Assuming this file is code which has been moved and parameterized rather than modified in meaningful ways, and thus not reviewing in detail.  Let me know if I need to take a closer look.